test/session.test.ts

```diff
@@ -922,49 +929,3 @@ describe("Prepared template tests", () => {
     );
   });
 });
-
```

Comment:
these tests where just copypaste of the previous ones. eslint started to complain about duplicate describes which is how i caught it